src/khoj/interface/web/index.html

```diff
@@ -188,9 +189,15 @@
     </script>
 
     <body>
-        <h1>
-            <img class="khoj-logo" src="static/assets/icons/khoj-logo-sideways.svg" alt="Khoj"></img>
-        </h1>
+        <!--Add Header Logo and Nav Pane-->
+        <div class="khoj-header">
+            <img class="khoj-logo" src="/static/assets/icons/khoj-logo-sideways.svg" alt="Khoj"></img>
```

Comment:
Does this asset need to be checked in?

Reply:
That's been already checked in a previous commit